tel2puml/find_unique_graphs/config.yaml

```diff
@@ -1,6 +1,53 @@
 data_sources:
   json:
-    path: /path/to/your/json/directory/or/file
+    # Specify the file path to the JSON data
+    dirpath: null
+    filepath: data.json
+    # Define the field mapping from the application to your JSON structure
+    data_location: resource_spans
+    header_mapping:
+      header:
+        key_paths: [resource:attributes::key, resource:attributes::key]
```

Comment:
Sorry to do this but I'm just thinking that we can have header as a collection of paths from the data e.g.
```yaml
header:
    paths: [resource:attributes, scope_spans::scope]
```
this would add those flattened fields to a dict which we could then access which could be put in the config e.g. for job name
```yaml
field_mapping:
      job_name:
        key_paths: [HEADER:resource:attributes::key]
        key_value: [service.name]
        value_paths: [value:Value:StringValue]
        value_type: string
```